GAMMA/.upgrade-assistant

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
I don't think this file is required - it's just a side effect of the upgrade, and I recommend we delete after review.